### ISSUE TITLE
Validation Losses- Reg

### ISSUE THREAD

**Issue by jyothsna-phd22** [open-mmlab/OpenLidarPerceptron#1658]
Hi,

I want to monitor validation losses along with training loss and plot both losses to check if the model is over fitting on kitti dataset. 

I request you to help me in this regard.

**Comment by jyothsna-phd22** [comment]
I am getting errors in model_func(...) method and also anchor_head_template.py(dimension issue) when writing code for validation loss.
